src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -174,16 +171,16 @@ public int getDefaultCloudIndexPrefetchBufferSize() {
     private List<ChromosomeInterval> intervals;
 
     // List of all sample names
-    private List<String> sampleNames = new ArrayList<>();
+    private final Set<String> sampleNames = new LinkedHashSet<>();
```

Comment:
Maybe we should just use `keySet()` from `sampleNameToVcfUri` instead of maintaining this `Set` separately